src/Networking/v2/Service.php

```diff
@@ -48,4 +48,42 @@ public function getNetwork($id)
     {
         return $this->model('Network', ['id' => $id]);
     }
+
+    /**
+     * Create a new subnet resource.
+     *
+     * @param array $options {@see \OpenStack\Networking\v2\Api::postSubnet}
+     *
+     * @return \OpenStack\Networking\v2\Models\Subnet
```

Comment:
If you're referencing the path namespace as an import path for this class, I don't see any harm in using the shorter version: `Subnet`. The FQCN is absolutely fine though :smile: